front/src/components/item/Issue.vue

```diff
@@ -3,6 +3,11 @@
     <h3 v-if="issue.repository">{{issue.repository.name}}</h3>
     <h4>{{issue.title}}</h4>
     <em v-html="marked(issue.body)"></em>
+    <div>
+      <span v-for="label in issue.labels">
```

Comment:
The :key attribute is missing. Is it normal ?

Reply:
![image](https://user-images.githubusercontent.com/3717296/46422714-ede6fb80-c702-11e8-9184-20b6f7aa7598.png)
 Fix in the next PR #33